example/plugins/c-api/protocol/Protocol.h

```diff
@@ -36,6 +36,12 @@
 
 #define PLUGIN_NAME "protocol"
 
+namespace protocol_ns
+{
+extern DbgCtl dbg_ctl;
+}
+using namespace protocol_ns;
```

Comment:
I know this is just an example, but putting this `using namespace` in a header might lead to user astonishment.

Reply:
I see your point, but what do you see as the better alternative?

TS developers must steel themselves to endure astonishment, and not in a good way.